src/server/helpers/error.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+/**
+ * Define error constants
+ */
+export const BAD_REQUEST = 400;
+export const CONFLICT = 409;
+export const NOT_FOUND = 404;
+export const GENERIC_ERROR = 500;
+
+/**
+ * Wrapper for javascript native Error() constructor
+ * 
+ * @param {String} message
+ * @param {number} status 
+ * 
+ * @returns {object} error
+ */
+export const createError = ({ message = 'Something went wrong, try again...', status = 500 }) => {
```

Comment:
This error could be a little more descriptive 🤔

Reply:
Noted, thanks.